bentoml/_internal/service/inference_api.py

```diff
@@ -39,7 +45,77 @@ def __init__(
         InferenceAPI._validate_route(route)
 
         self.name = name
+        self.needs_ctx = False
+        self.ctx_param = None
         self.func = user_defined_callback
+        input_type = input_descriptor.input_type()
+        self.multi_input = isinstance(input_type, dict)
+        sig = inspect.signature(user_defined_callback)
+        if isinstance(input_type, dict):
+            # note: in python 3.6 kwarg order was not guaranteed to be preserved,
+            #       though it is in practice.
+            for key in input_type:
+                if key not in sig.parameters:
+                    if (
+                        key in ["context", "ctx"]
+                        or sig.parameters[key].annotation == bentoml.Context
```

Comment:
we should import `Context` from `..context` instead of `import bentoml`

Reply:
I think for public facing API, we should just use the public facing import path